test/lib/next-modes/next-dev.ts

```diff
@@ -88,7 +88,8 @@ export class NextDevInstance extends NextInstance {
                 msg,
               })
             }
-            this.off('stdout', readyCb)
+            // dev server can reload on next.config.js change and that gets a new port so we want to keep listening and update if needed
+            // this.off('stdout', readyCb)
```

Comment:
remove?

Reply:
I wanted to leave there a comment that we actually want to keep listening which is not clear, but I can remove that snippet.